language-design.md

```diff
@@ -70,6 +205,12 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
+The language stereotypes explored in the Yang and Rabkin article are very much applicable to the use of DSL’s. People who use DSL’s are susceptible to this same “language-based snobbery”. Consider the example of jQuery. In one of Julien’s previous internships, jQuery was scoffed at – although there were performance-related improvements to its eventual removal. Even though it is widely used and no doubt simplifies interactions with HTML, some programmers may think of it as a crutch, and a sign of weakness as it removes the need to understand the DOM on a deeper level. 
```

Comment:
It is very frequently helpful to understand how something works under the hood. You don't necessarily need to know all of the dirty details (jQuery has lots of tricks to the implementation) but at least a superficial understanding makes it easier to use the tool effectively.